steam-buddy

```diff
@@ -6,12 +6,15 @@ import yaml
 import json
 import shutil
 import subprocess
-import flathub
-from glob import glob
+from steam_buddy.flathub import Flathub
 from bottle import route, request, static_file, run, template, redirect, response
 
 DATA_DIR = os.getenv('XDG_DATA_HOME', os.path.expanduser('~/.local/share'))
 
+RESOURCE_DIR = os.getcwd()
+if not os.path.isdir(os.path.join(RESOURCE_DIR, 'views')):
```

Comment:
A little worried that this might not be specific enough of a test. I could imagine a command called `views` existing under `/usr/bin`. What if we looked for a specific file?

Reply:
It will only return ``True`` in case of a directory. I could maybe match more directories here, but I don't think that would be needed. I don't want to rely to much on paths which could change in the future. The nice thing about the views directory is that bottle requires it to be called that.

Reply:
Actually, I could check it the other way around and only move the resource dir if the location of the script we're running is /usr/bin

Reply:
Nevermind, I made it check for a specific view file instead now. Detecting the location of the current script didn't really seem reliable enough.